### PR TITLE
Classes, and asynchronous support.

### PR DESCRIPTION
**Pull request by lamaqdahodwala**
We now have a better organization system for the API, including the bases of async support. This also opens the door for people to use OOP on the API, creating their own classes to do whatever. This also allows for people to have multiple REPLAPI instances at the same time.

### CODE REVIEW

File: App/REPLAPI/main.py [hunk 1]
```diff
@@ -11,131 +15,182 @@
   print(cont)
 '''
 
+class User():
+  def __init__(self, user):
+    self.name = user
```
Comment (lamaqdahodwala) [inline App/REPLAPI/main.py:20]: All the code here is the same, the new part is at the async User

**Comment by darkdarcool** [comment]
we are just starting to use those for the next version!(except classes for this, we will organize everything when we are going to stop supporting replapi)

**Comment by lamaqdahodwala** [comment]
Sounds good. Should we continue with asynchronous support?

**Comment by darkdarcool** [comment]
i haven't looked over your pull rest very well, so ill get back to you on that

**Comment by darkdarcool** [comment]
when I look over it, we are doing it similary, but in a better way, but thanks for the idea(our way uses shorter code that is faster and sent use json, and uses async that the user doesn't have to use await in a function on `0`-`0`)

**Comment by lamaqdahodwala** [comment]
Really? Well that sounds big brain. I'll let u get to work

**Comment by darkdarcool** [comment]
thanks! if you go into main.py and look at the assets class, you can see how we did it 0-0

**Pull request closed without merging**